packages/react-dev-utils/getCSSModuleLocalIdent.js

```diff
@@ -23,7 +23,7 @@ module.exports = function getLocalIdent(
     : '[name]';
   // Create a hash based on a the file location and class name. Will be unique across a project, and close to globally unique.
   const hash = loaderUtils.getHashDigest(
-    context.resourcePath + localName,
+    context.resourcePath.replace(context.rootContext, '') + localName,
```

Comment:
This breaks in subtle ways. I think you should use something more safe, like `path.posix.relative(context.rootContext, context.resourcePath)`.

Reply:
thanks for the recommendation, I'll make this change and push an update.